packages/terminal-extension/src/index.ts

```diff
@@ -141,17 +147,23 @@ function activate(
       console.error(reason.message);
     });
 
+  // Subscribe to changes in theme.
+  themeManager.themeChanged.connect((sender, args) => {
+    tracker.forEach(widget => {
+      const terminal = widget.content;
+      terminal.setOption('theme', args.newValue as 'light' | 'dark');
```

Comment:
Isn't `args.newValue` the name of whatever theme the use might have loaded? Why cast it to `'light' | 'dark'`?

The theme manager provides a test function for whether a given theme is largely light or dark:
https://github.com/jupyterlab/jupyterlab/blob/a99cfda0466fd6cf15a77b434e66811c1ec2c53c/packages/apputils/src/thememanager.ts#L147-L152